Programing-Paradigm-And-Methodology.md

```diff
@@ -0,0 +1,18 @@
+# Programing Paradigm and Methodology
+## Foundational Agile Principles
+**_Individuals and interactions_** over processes and tools
+
+**_Working software _**over comprehensive
+
+_**Customer collaboration**_ over contract negotiation
+
+_**Responding to change **_ over following a plan
+
+## User-Stories
+User stories help us tie the needs of our users directly to features we implement in a conversational format. They **always** follow this syntax: _"As a User/Stakeholder , I need/want to be able to Do Something"_. This can result in some awkward stories for the "obvious" features like _"As a user, I need to be able to trust that my credit card information is secure."_, but it is imperative that ALL features tie back to a story in order to allow us to effectively break the work down into manageable pieces.
```

Comment:
*...User/Stakeholder , I need/want to be ab...*

the comma should be attached with *Stakeholder,*